WordPress/Classes/ViewRelated/NUX/SignupEpilogueTableViewController.swift

```diff
@@ -183,29 +183,14 @@ private extension SignupEpilogueTableViewController {
             if let customDisplayName = dataSource?.customDisplayName {
                 userInfo.fullName = customDisplayName
             } else {
-                let autoDisplayName = generateDisplayName(from: userInfo.email)
+                let autoDisplayName = Self.generateDisplayName(from: userInfo.email)
```

Comment:
Made this `static` because the method does not depend on any instance state and it makes it simpler to test.

Really, extracting a WordPress.com display name from a `String`/email is orthogonal to the view layer. It could have been extracted in a dedicated type or added to an already existing type with similar utils. However, given the note about the duplication with `ZendeskUtils` and the fact that this is a just-in-case bugfix, I decided to leave it here for the moment.

Keen to hear what you think.